uidisplay.m

```diff
@@ -442,5 +474,152 @@ int main(int argc, char *argv[]) {
+					break;
+				default:
+					printJSON(@[
+						@{
+							@"autoBrightness": stateAsJSONType(getAutoBrightness()),
+							@"displayName": [NSString stringWithUTF8String:_("Auto-Brightness")]
+						},
+						@{
+							@"brightness": @(getBrightness()),
+							@"displayName": [NSString stringWithUTF8String:_("Brightness")]
+						},
+						@{
+							@"darkMode": stateAsJSONType(getDarkMode()),
+							@"displayName": [NSString stringWithUTF8String:_("Dark Mode")]
+						},
+						@{
+							@"nightShift": stateAsJSONType(getNightShift()),
+							@"displayName": [NSString stringWithUTF8String:_("Night Shift")]
+						},
+						@{
+							@"trueTone": stateAsJSONType(getNightShift()),
+							@"displayName": [NSString stringWithUTF8String:_("True Tone")]
+						},
+						@{
+							@"reduceWhitePoint": @(_AXSReduceWhitePointEnabled()),
+							@"displayName": [NSString stringWithUTF8String:_("Reduce White Point")]
+						},
+						@{
+							@"height": @(CGRectGetHeight([screen bounds])),
+							@"displayName": [NSString stringWithUTF8String:_("Height")]
+						},
+						@{
+							@"width": @(_AXSReduceWhitePointEnabled()),
+							@"displayName": [NSString stringWithUTF8String:_("Width")]
+						},
+						@{
+							@"scale": @(_AXSReduceWhitePointEnabled()),
+							@"displayName": [NSString stringWithUTF8String:_("Scale")]
+						}
+					]);
+					break;
+			}
+		}
+		else {
+			switch (info) {
+				case iAutoBrightness:
+					printf(_("Auto-Brightness: %s\n"), stateAsString(getAutoBrightness()));
+					break;
+				case iBrightness:
+					printf(_("Brightness: %.6g\n"), getBrightness());
+					break;
+				case iDarkMode:
+					printf(_("Dark Mode: %s\n"), stateAsString(getDarkMode()));
+					break;
+				case iNightShift:
+					printf(_("Night Shift: %s\n"), stateAsString(getNightShift()));
+					break;
+				case iTrueTone:
+					printf(_("True Tone: %s\n"), stateAsString(getTrueTone()));
+					break;
+				case iReduceWhitePoint:
+					printf(_("Reduce White Point: %s\n"), stateAsString(_AXSReduceWhitePointEnabled() ? sOn : sOff));
+					break;
+				case iHeight:
+					printf(_("Height: %f\n"), CGRectGetHeight([screen bounds]));
+					break;
+				case iWidth:
+					printf(_("Width: %f\n"), CGRectGetWidth([screen bounds]));
+					break;
+				case iScale:
+					printf(_("Scale: %f\n"), [screen scale]);
+					break;
+				default:
+					printf(_("Brightness: %.6g\n"), getBrightness());
+					printf(_("Auto-Brightness: %s\n"), stateAsString(getAutoBrightness()));
+					printf(_("Dark Mode: %s\n"), stateAsString(getDarkMode()));
+					printf(_("Night Shift: %s\n"), stateAsString(getNightShift()));
+					printf(_("True Tone: %s\n"), stateAsString(getTrueTone()));
+					printf(_("Reduce White Point: %s\n"), stateAsString(_AXSReduceWhitePointEnabled() ? sOn : sOff));
+					printf(_("Height: %f\n"), CGRectGetHeight([screen bounds]));
+					printf(_("Width: %f\n"), CGRectGetWidth([screen bounds]));
+					printf(_("Scale: %f\n"), [screen scale]);
```

Comment:
Use the some strings here and for json so that we don't have duplicate strings to be translated. Like this:
```c
printf("%s: %s\n", _("Auto-Brightness"), getAutoBrightness());
```